### ISSUE TITLE
Change the order in the first level from "remotes" into the correct one

### ISSUE THREAD

**Issue by ITaluone** [pcottle/learnGitBranching#1169]
Hi

I do like this project very much, thanks for that! 👍

But let me nitpick a litte ;). What I think about is to change the order of `git clone`, meaning starting with the remote repo "somewhere" and make your local copy with `git clone`.

I think this should be doable (at least) and would help a lot to grasp a better understanding of the clone command itself (in real world). For example: a clone command always takes a parameter which points to the URL (in most cases) of the remote.

So what I am thinking of is, that the remote repo lives somewhere out there (+ has a URL-like location and/or a name), which have to be included in the clone command e.g. `git clone location/reponame.git`

This would help a lot, because this is 1:1 how it works in real.

But if this is a major redesign, I am good about leaving as is :)

**Comment by pcottle** [comment]
I totally agree that git clone in the real world operates like that, not like how it's depicted in our lessons 😅

Unfortunately when I started this project, I didn't plan on doing remote lessons at all! So the ability to git clone was a bit of an afterthought and I did the easiest option based on how the codebase was built -- which required always having a local git tree -- even if it was inaccurate. I figured you still get all the valuable learning from the lessons even if the setup is inaccurate.

Anyways like you said, it's a bit project to change this so probably not going to be done anytime soon. Just wanted to say I agree with you and it's a minor regret of mine from doing this project. It's come up in other issues as well!

Thanks for stopping by and leaving the feedback though
